public/locales/de/_defaults.json

```diff
@@ -2,5 +2,6 @@
     "Password": "Passwort",
     "Select action": "Aktion wählen",
     "What would you like to do?": "Was möchtest Du tun?",
-    "{{error}}, please wait {{time}} seconds.": "{{error}}, bitte warte {{time}} sekunden."
+    "{{error}}, please wait {{time}} seconds.": "{{error}}, bitte warte {{time}} sekunden.",
+    "Connection Lost": "Verbindung unterbrochen"
```

Comment:
`Connection Lost` vs `Connection lost` -- capital L vs lowercase L...
And while you're at it: Capital `S` in the line above for "Sekunden"